Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -120,6 +174,82 @@ public static string[] GetLanguages()
             return localizationTable.Keys.ToArray();
         }
 
+        public static void LoadConfigFile(string pathToConfigFile)
+        {
+            localizationConfigurations = new Dictionary<string, LanguageData>();
+
+            if (File.Exists(pathToConfigFile) == false)
+            {
+                Debug.ULogErrorChannel("LocalizationTable", "No config file found at: " + pathToConfigFile);
```

Comment:
Well what should happen is that once it is downloaded, everything should work as normal and the localization should reload itself, but that doesnt seem happening.